agrotoxico/serializers/agrotoxico.py

```diff
@@ -14,3 +15,11 @@ class Meta:
         model = Agrotoxico
         fields = ('idAgrotoxico', 'nome', 'tipo')
         read_only_fields = ('idAgrotoxico',)
+
+
+class AgrotoxicoListSerializer(serializers.ModelSerializer):
+    tipo = TipoAgrotoxicoSerializer()
+
+    class Meta:
+        model = Agrotoxico
+        fields = ('idAgrotoxico', 'nome', 'tipo')
```

Comment:
Não é necessário criar um novo serializer com os mesmos campos do AgrotoxicoCreateSerializer, pode usar ele. Vendo aqui parece que só o tipo que está sendo serializado em um outro nível, se isso for realmente necessário, faz a herança de serializers e apenas adiciona esse novo campo.
Também não esqueça do read_only